doc/doxygen/stylesheet.css

```diff
@@ -26,3 +26,11 @@ div.doxygen-generated-exception-message {
     padding: 5px;
     font-size: 95%;
 }
+
+div.image img[src="fe_enriched_1d.png"] {
+    width:500px;
```

Comment:
oh, that seems like the wrong place to adjust the width (because it's going to be impossible to find if anyone ever wonders where the width is set, and because it will definitely be forgotten to remove this text if the figure is ever removed or renamed). Please attach this (and the next) width directly to the place where you have the `@img` tag.